docs/howto/how-to-build-package.md

```diff
@@ -1 +1,169 @@
 # How to Build Package
```

Comment:
How about using `nnpackage` instead of just `package`?

Reply:
This was set when created. If we use `package`, it would be better to use `How to build package`. But as this is a title, `Build Package` is OK I think.